src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -22,6 +22,8 @@ import { useFormAvailableUseCases } from '../workspace_form/use_form_available_u
 import { NavigationPublicPluginStart } from '../../../../../plugins/navigation/public';
 import { DataSourceConnectionType } from '../../../common/types';
 import { WorkspaceCreatorForm } from './workspace_creator_form';
+import { DATA_CONNECTION_SAVED_OBJECT_TYPE } from '../../../../../plugins/data_source/common/data_connections';
```

Comment:
same as above with using the index and combining these two

Reply:
Updated.